src/WebJobs.Extensions.DurableTask/Bindings/OrchestrationTriggerAttributeBindingProvider.cs

```diff
@@ -20,6 +20,7 @@ internal class OrchestrationTriggerAttributeBindingProvider : ITriggerBindingPro
     {
         private readonly DurableTaskExtension config;
         private readonly string connectionName;
+        private static readonly JsonSerializer jsonSerializer = JsonSerializer.Create();
```

Comment:
Seems the tests are failing because this field does not begin with an upper-case letter. Mind tweaking this?

Reply:
> Seems the tests are failing because this field does not begin with an upper-case letter. Mind tweaking this?

Done

> Have you validated manually that this fixes the problem when in combination with WebPubSub?

Yep!